projects/authentication/docs/user-access-token.md

```diff
@@ -15,7 +28,7 @@ See [requesting client credentials](./requesting-credentials.md) how to obtain a
 
 Additionally, we will need to configure the following settings for your client on our end:
```

Comment:
```suggestion
Additionally, you will need to configure the following settings for your client on [Publiq Platform](https://platform.publiq.be/). Select your integration and select the "settings" tab.
```

Reply:
I think this is partially context dependent. Not familiar enough with the UDB context here, but for UiTPAS, the user permissions will have to be set up in the UiTPAS Admin. At this point, this isn't possible on publiq platform.

Are user permissions even a thing in UDB? This might be related to roles (and that would make sense to align it, so UPS and UDB have a similar logic). But this is out of scope of this request.

Reply:
User permissions are managed directly in UDB, specifically there is a list of emails that are the admins.
The text below is referring to login, callback and logout urls, and those are managable in Publiq Platform.
Maybe we should just move this outside this PR and create a seperate ticket for this, it is not directly related to Keycloak.

Reply:
> Maybe we should just move this outside this PR and create a seperate ticket for this, it is not directly related to Keycloak.

yes please 🙏 
I don't think I changed that line :)

Reply:
I made a ticket https://jira.publiq.be/browse/PPF-638 so ignore this remark for now.